rest_framework_json_api/utils.py

```diff
@@ -236,15 +243,11 @@ def get_related_resource_type(relation):
                     relation_model = parent_model_relation.related.related_model
                 else:
                     relation_model = parent_model_relation.related.model
-            elif parent_model_relation_type is ManyToManyDescriptor:
```

Comment:
My general strategy was to apply the changes this made to the new codebase (ie, a manual rebase).

I don't recall what my logic is here, but it does look like a merge failure.